### DISCUSSION TITLE
How to add payload for ZB-SW02 eWelink

### DISCUSSION THREAD

**Discussion by AkLim94** [Koenkk/zigbee2mqtt#20997] [Devices]
Hi,
I use the ZB-SW02 in Inching mode. I saw in the documentation it’s possible to extend the maintain time by adding a parameter in payload (below axtract from doc):
On with timed off
When setting the state to ON, it might be possible to specify an automatic shutoff after a certain amount of time. To do this add an additional property on_time to the payload which is the time in seconds the state should remain on. Additionnaly an off_wait_time property can be added to the payload to specify the cooldown time in seconds when the switch will not answer to other on with timed off commands. Support depend on the switch firmware. Some devices might require both on_time and off_wait_time to work Examples : {"state" : "ON", "on_time": 300}, {"state" : "ON", "on_time": 300, "off_wait_time": 120}

But I don’t know how and where to made this change 😩
Please can somebody can help me? Or may be just to point out where I can get some z2m documentation to perform that?

Thanks in advance
Regards

**Comment by AkLim94** [comment]
I found the json file related to this device:

![image](https://github.com/Koenkk/zigbee2mqtt/assets/118060168/ac73d998-a84f-4c3b-9e3f-854b8658426a)

But what to change in it?
Any help is welcome :)

**Comment by AkLim94** [comment]
Found the way to add payload

**Reply by herrphil99** [reply]
Can you please share it? I can't find it.

**Reply by olgru** [reply]
Me too...

**Reply by JaCh0o** [reply]
Also here... any solution ?
